docs/UserGuide.md

```diff
@@ -63,22 +63,43 @@ Additionally, each row of the csv file must contain exactly 4 commas (`,`). Opti
 
 You can consult this [mismatch file](exampleMismatchFile.csv) for a valid example.
 
-_**Note**: If possible, please do not include mismatches in your import file that have already been uploaded and reviewed before to avoid double-work by reviewers.
+**Note**: If possible, please do not include mismatches in your import file that have already been uploaded and reviewed before to avoid double-work by reviewers.
 
 ### Uploading an import file
 
-To upload an import file, users may send a request to our `POST /api/imports` api endpoint. 
+To upload an import file, users may send a request to our `POST /api/imports` API endpoint.
 
-The request should include the `Authorization` header with a personal [access token](#apiAccess). Additionally, the request should include a `Content-Type: multipart/form-data` header.
```

Comment:
Hello, I am not sure why this part was removed, this is needed when testing with Postman. @lydiapintscher @marfox